pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -1343,8 +1348,9 @@ func TestPreempt(t *testing.T) {
 		{
```

Comment:
make sure we have a test case that ensures a pod with a nil preemption policy behaves correctly